jumpcloud-ADMU/Powershell/Form.ps1

```diff
@@ -385,7 +398,7 @@ $lbNetBios.Content = $NetBiosName
 #AzureADInformation
 $lbAzureAD_Joined.Content = $AzureADStatus
 $lbTenantName.Content = $TenantName
-
+$hostname = hostname
```

Comment:
we've been using `$env:computername` for every other instance let's keep things consistent on how we get the hostname. 
```suggestion
$hostname = $env:computername
```